onnxmltools/convert/sklearn/operator_converters/LinearClassifier.py

```diff
@@ -66,7 +66,7 @@ def convert_sklearn_linear_classifier(scope, operator, container):
         normalized_probability_tensor_name = probability_tensor_name
 
     # Post-process probability tensor produced by LinearClassifier operator
-    if len(class_labels) > 2 or op.__class__.__name__ != 'LinearSVC':
+    if len(class_labels) > 2 and op.__class__.__name__ != 'LinearSVC':
```

Comment:
Why do we need to use `and`? `LinearSVC` has no probability output no matter how many classes do we have. Therefore, probability normalization in the else-branch below should be avoided. If this change is necessary, I'd suggest to open another PR to isolate unrelated fixes.